2022-11-24 02:26:35,600 INFO    MainThread:1667739 [wandb_setup.py:_flush():68] Configure stats pid to 1667739
2022-11-24 02:26:35,600 INFO    MainThread:1667739 [wandb_setup.py:_flush():68] Loading settings from /home/hnguyen/.config/wandb/settings
2022-11-24 02:26:35,600 INFO    MainThread:1667739 [wandb_setup.py:_flush():68] Loading settings from /home/hnguyen/cloth_smoothing/SAC/wandb/settings
2022-11-24 02:26:35,600 INFO    MainThread:1667739 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-24 02:26:35,600 INFO    MainThread:1667739 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'curl/run_curl.py', 'program': 'curl/run_curl.py'}
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:_log_setup():476] Logging user logs to /home/hnguyen/cloth_smoothing/SAC/wandb/run-20221124_022635-3ee25t2n/logs/debug.log
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:_log_setup():477] Logging internal logs to /home/hnguyen/cloth_smoothing/SAC/wandb/run-20221124_022635-3ee25t2n/logs/debug-internal.log
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:init():516] calling init triggers
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:init():569] starting backend
2022-11-24 02:26:35,601 INFO    MainThread:1667739 [wandb_init.py:init():573] setting up manager
2022-11-24 02:26:35,609 INFO    MainThread:1667739 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-24 02:26:35,614 INFO    MainThread:1667739 [wandb_init.py:init():580] backend started and connected
2022-11-24 02:26:35,622 INFO    MainThread:1667739 [wandb_init.py:init():658] updated telemetry
2022-11-24 02:26:35,652 INFO    MainThread:1667739 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2022-11-24 02:26:35,663 ERROR   MainThread:1667739 [wandb_init.py:init():1105] error
Traceback (most recent call last):
  File "/home/hnguyen/miniconda3/envs/softgym/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1078, in init
    run = wi.init()
  File "/home/hnguyen/miniconda3/envs/softgym/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 696, in init
    handle = backend.interface.deliver_run(run)
  File "/home/hnguyen/miniconda3/envs/softgym/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 719, in deliver_run
    run = self._make_run(run_obj)
  File "/home/hnguyen/miniconda3/envs/softgym/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 173, in _make_run
    run._make_proto_run(proto_run)
  File "/home/hnguyen/miniconda3/envs/softgym/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 664, in _make_proto_run
    run.display_name = self._name
TypeError: 0 has type int, but expected one of: bytes, unicode
2022-11-24 02:26:38,866 WARNING MsgRouterThr:1667739 [router.py:message_loop():77] message_loop has been closed
